crates/ruma-state-res/benches/state_res_bench.rs

```diff
@@ -631,7 +631,7 @@ mod event {
     }
 
     #[derive(Clone, Debug, Deserialize, Serialize)]
-    pub struct StateEvent {
+    pub struct OriginalStateEvent {
```

Comment:
Was this a find/replace freebie/mistake? It doesn't matter since it's just for the tests, but I wonder if, at some point, it should be called `PduEvent`?